packages/logger/src/rld-filter.ts

```diff
@@ -0,0 +1,148 @@
+import { Action, Atom, AtomMaybe, Ctx, action, atom } from '@reatom/core'
+import { h } from '@reatom/jsx'
+import styled from 'stylerun'
+import * as model from './rld-filter-model'
+import { t } from './t'
+import { match } from '@reatom/lens'
+import { buttonStyles } from './rld'
+
+export function RldFilterMenu() {
+  return atom((ctx) =>
+    t.div({
+      ...menuStyles({}),
+      children: [
+        RldFilterMenuDraft(),
+        match(ctx.get(model.filters).length, t.div(draftHrStyles({}))),
```

Comment:
Did you forget the `spy`?
```suggestion
        match(ctx.spy(model.filters).length, t.div(draftHrStyles({}))),
```

You could put the computation as a function to the match method to scope reactivity to the match atom (prevent rerenders).
```suggestion
        match(ctx => ctx.spy(model.filters).length, t.div(draftHrStyles({}))),
```